dependencies.gradle

```diff
@@ -3,6 +3,7 @@ dependencies {
     compile("com.github.GTNewHorizons:StructureLib:1.0.15:dev")
     compile("net.industrial-craft:industrialcraft-2:2.2.828-experimental:dev")
     compile("com.github.GTNewHorizons:AppleCore:3.1.9:dev")
+    compile("com.github.GTNewHorizons:ProjectRed:master-SNAPSHOT:dev")
```

Comment:
Do you really need to pull ProjectRed? Wouldn't `compile("mrtjp:MrTJPCore:1.7.10-1.1.0.33:dev"` suffice?

Reply:
No snapshot Masters.  Pick a tag like glow said

Reply:
Fixed by using MrTJPCore as glow suggested. Regarding the snapshot issue, I used [this gradle script](https://github.com/GTNewHorizons/ProjectBlue/blob/master/dependencies.gradle#L4) as a reference so I don't know if that needs updating.

Reply:
basically, by using snaptshots from master, you compile with the latest code availiable on master. That sounds interesting when you don't want to update the deps in the build script often, but it's actually a bad idea, as your builds become unreproductible.

Reply:
We haven't updated all repos, the one you linked is indeed out of date.